sdk/identity/identity/Troubleshooting.md

```diff
@@ -1,48 +1,85 @@
+```ts
+import @azure/identity
+import @azure/keyvault-secrets
+
+// Create a key client using the DefaultAzureCredential
+ const keyVaultUrl = `https://key-vault-name.vault.azure.net`;
+ const client = new KeyClient(keyVaultUrl, new DefaultAzureCredential());
+
+ try {
+// Retrieving the properties of the existing keys in that specific Key Vault.
+    console.log(await client.listPropertiesOfKeys().next());
+ } catch (error) {
+    console.log("Authentication Failed", error.message);
+ }
+```
+
+### CredentialUnavailableError
+
+The `CredentialUnavailableError` is used to indicate that the credential can’t authenticate in the current environment, due to lack of required configuration or setup. This error is also used as a signal to chained credential types, such as `DefaultAzureCredential` and `ChainedTokenCredential`, that the chained credential should continue to try other credential types later in the chain. In `ManagedIdentityCredential` it can also trigger if the authentication endpoints (like the IMDS endpoint) is unavailable.
+
+### AuthenticationError
+
+The `AuthenticationError` is used to indicate about a failure to authenticate with Azure Active Directory. The `errorResponse` field contains more details about the specific failure.
+
+## Finding Relevant Information in Error Messages
 
-Follow the configuration instructions in the "Credential unavailable error" sections of the troubleshooting guidelines below for the respective credential/authentication type you're looking to use via `DefaultAzureCredential`:
+`AuthenticationRequiredError` is thrown when unexpected errors occurred while a credential is authenticating. This can include errors received from requests to the AAD STS and often contains information helpful to diagnosis. Consider the following `AuthenticationRequiredError` message.
 
-| Credential Type               | Troubleshooting Guide                                                                      |
-| ----------------------------- | ------------------------------------------------------------------------------------------ |
-| Environment credential        | [Environment credential guide](#troubleshoot-environment-credential-authentication-issues) |
-| Managed Identity credential   | [Managed Identity guide](#troubleshoot-managed-identity-authentication-issues)             |
-| Visual Studio Code credential | [Visual Studio Code guide](#troubleshoot-visual-studio-code-authentication-issues)         |
-| Azure CLI credential          | [Azure CLI guide](#troubleshoot-azure-cli-authentication-issues)                           |
-| Azure PowerShell credential   | [Azure PowerShell guide](#troubleshoot-azure-powershell-authentication-issues)             |
+<--TODO-->
```

Comment:
Is this intentional?

Reply:
Yes need to add in an example here. The image was missing from Scott's PR, so kinda clueless what I need to add here.

Reply:
added an example